test/dynamics/common.py

```diff
@@ -82,4 +98,21 @@ def jit_grad_wrap(self, func_to_test: Callable) -> Callable:
             JIT-compiled gradient of function."""
         wf = wrap(lambda f: jit(grad(f)), decorator=True)
         f = lambda *args: np.sum(func_to_test(*args)).real
-        return wf(wrap(f))
+        return wf(f)
+
+
+class TestQutipBase(unittest.TestCase):
+    """Base class with setUpClass and tearDownClass for setting jax as the
+    default backend.
+
+    Test cases that inherit from this class will automatically work with jax
+    backend.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            # pylint: disable=import-outside-toplevel
+            import qutip
```

Comment:
Just disable all pylint warnings associated with this import. I really don't think there's anything else that can be done.